geti_sdk/deployment/deployed_model.py

```diff
@@ -489,99 +489,6 @@ def _postprocess(
         """
         return self._inference_model.postprocess(inference_results, metadata)
 
-    def _postprocess_explain_outputs(
```

Comment:
Great that we can get rid of this, I think it was still a remnant from before the OTX models were fully switched to model API.